Fix ACMD41. Closes #5

richardeoin opened the pull request:
* Restrict bitwise voltage window to valid values only
* Name the function and its arguments using words, not a jumble of letters

richardeoin committed:
Fix ACMD41. Closes #5

* Restrict bitwise voltage window to valid values only
* Name the function and its arguments using english words, not a jumble of
letters

jkristell commented:
Thanks for debugging this. I shouldn't be trusted with bitmasks.

Perhaps we should define a VoltageWindow helper that could be used here and in the Ocr structure.

richardeoin committed:
Specify voltage window for sd_send_op_cond as 9-bit bitfield

richardeoin commented:
@jkristell I've rebased this onto master. 

I think a helper function that creates the bitfield from a voltage range is probably overkill here, since the value would usually be hardcoded. Instead I've added some docs to help most users through the default case.

jkristell commented:
Thanks!

I can do a 0.7 with this later today.
